# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1327431729 -3600
#      Tue Jan 24 20:02:09 2012 +0100
# Branch gui
# Node ID 605675153f1d08029a8248ce147670da49b5252f
# Parent  d19e3954c3152d6ef2ddd4e19b40bf59ee439d55
Added similar script to retrieve libqirc.

diff --git a/gui/get-qirc.sh b/gui/get-qirc.sh
new file mode 100755
--- /dev/null
+++ b/gui/get-qirc.sh
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+qirc_dir="qirc";
+echo "Updating qirc..";
+
+# Check whether we have already cloned the repository:
+if [ -d $qirc_dir ]; then
+	# Yes, so just pull any changes and rebuild.
+	cd qirc
+	git pull
+	qmake-qt4 qirc.pro
+	cd ..
+else
+	# No, clone the repository, checkout the stable branch
+	# and build it.
+	git clone https://code.google.com/p/qirc/
+	git checkout master
+	cd qirc
+	qmake-qt4 qirc.pro
+	make
+	cd ..
+fi
diff --git a/gui/get-qterminal.sh b/gui/get-qterminal.sh
--- a/gui/get-qterminal.sh
+++ b/gui/get-qterminal.sh
@@ -3,17 +3,17 @@
 qterminal_dir="qterminal";
 echo "Updating qterminal..";
 
 # Check whether we have already cloned the repository:
 if [ -d $qterminal_dir ]; then
 	# Yes, so just pull any changes and rebuild.
 	cd qterminal
 	git pull
-	qmake-qt4
+	qmake-qt4 qterminal.pro
 	cd ..
 else
 	# No, clone the repository, checkout the stable branch
 	# and build it.
 	git clone https://code.google.com/p/qterminal/
 	git checkout master
 	cd qterminal
 	qmake-qt4 qterminal.pro
